lib/src/importer-registry.ts

```diff
@@ -11,6 +11,13 @@ import {FileImporter, Importer, Options} from './vendor/sass';
 import * as proto from './vendor/embedded_sass_pb';
 import {catchOr, thenOr, PromiseOr} from './utils';
 
+export class NodePackageImporter {
+  entryPointPath?: string;
```

Comment:
This shouldn't be a public field—since it can't be private, it might be best to make it a local symbol.

Reply:
Added in https://github.com/oddbird/embedded-host-node/commit/b8e96d64c1343d73d3ab053c00f5c8b3f94b77bf